curvefs/src/client/inode_cache_manager.h

```diff
@@ -168,8 +161,8 @@ class InodeCacheManagerImpl : public InodeCacheManager {
     std::map<uint64_t, std::shared_ptr<InodeWrapper>> dirtyMap_;
     curve::common::Mutex dirtyMapMutex_;
 
-    // inodeid to parent inodeid, may have more parent at hard link
-    std::map<uint64_t, std::list<uint64_t>> parentIdMap_;
+    // inodeid to parent inodeid
+    std::map<uint64_t, uint64_t> parentIdMap_;
```

Comment:
Does this only work if you turn it on?

Reply:
> Does this only work if you turn it on?

Yes, the relationship is needed when turn on the enableSumInDir flag.